sdk/iOS/OoyalaSkinSDK/ViewControllers/OOSkinViewController.h

```diff
@@ -28,7 +28,13 @@ extern NSString *const OOSkinViewControllerFullscreenChangedNotification; /* Fir
 /**
  Programatically change the fullscreen mode of the player.
  */
-@property(nonatomic, getter=isFullscreen) BOOL fullscreen;
+@property (nonatomic, getter=isFullscreen) BOOL fullscreen DEPRECATED_MSG_ATTRIBUTE("Setter is deprecated and will be removed in a future version. Use autoFullscreenWithRotatedEnabled instead for enable/disable auto full screen mode after device rotation changes.");
```

Comment:
why deprecate the setter? For uses cases in which you just want to go fullscreen by another action which is not rotation it should still be valid, isn't it? It's ok if we add another property (`autoFullscreenWithRotatedEnabled`) just to deal with the fullscreen when rotated.

Reply:
The default full screen logic may be working not correct with iOS rotation animation, because we can't determine what the full screen animation was initialized with. (caused by rotation or just setter call).

So we can save this implementation, but we can't guarantee a correct animation when customers set the full screen when the device did change orientation.

Also we can add an additional method like this:
```- (void)setFullscreen:(BOOL)fullscreen isCausedByRotation:(BOOL)isCausedByRotation```

Reply:
remove the `DEPRECATED_MSG_ATTRIBUTE` for now, we won't remove this setter but will encourage customers to use the new one to deal with fullscreen when rotating.